test/examples/break1.rb

```diff
@@ -4,6 +4,10 @@
 # A toy example for testing break commands.
 #
 class Break1Example
+  undef a if method_defined? :a
+  undef b if method_defined? :b
+  undef c! if method_defined? :c!
+
```

Comment:
Can you think of a way of not having to do this? I imagine we could leave only the class definition in this file and require it at the beggining of the tests that use it. And then explicitly run

    binding.pry
    Break1Example.new.a

for each test.

The reason I'd like not to add this is that it feels out of place and it has nothing to do with the test...

In any case, if you'd rather not work on this or you don't agree, just add a TODO comment on top of it (so that I remember to give this another thought) and I'll merge as it is.

Thanks!